spec/controllers/accounts/registrations_controller_spec.rb

```diff
@@ -1,30 +1,36 @@
-require 'spec_helper'
+require "spec_helper"
 
 describe Accounts::RegistrationsController do
   it "can get signup form" do
     get :new
     expect(response).to be_success
   end
 
-  it 'can signup (create action)' do
-    expect do
-      post :create, :user => { :username => "signup_username", :name => "Mr. SignUp", :email => "signup@nztrain.com", :password => "password", :password_confirmation => "password" }
-    end.to change{User.count}.by(1)
+  it "can signup (create action)" do
+    expect {
+      post :create,
+        user: {username: "signup_username", name: "Mr. SignUp", email: "signup@nztrain.com", password: "password",
+               password_confirmation: "password"}
+    }.to change { User.count }.by(1)
     # check signup attributes saved
     newuser = User.find_by_username("signup_username")
     expect(newuser).not_to be_nil
     expect(newuser.name).to eq("Mr. SignUp")
     expect(newuser.email).to eq("signup@nztrain.com")
     expect(newuser.valid_password?("password")).to be true
+
+    # Due to how transactions are used in tests under Rails < 5
+    # these tests don't work on modern devise (fixed / broken in 4.1.0
+    # TODO: Re-enable these lines after we're on rails 5
```

Comment:
This can be removed now (in which case the rest of the changes to this file are formatting changes only).

Sidenote: I'm kinda keen to just run `standardrb --fix` over the whole codebase and commit those changes in a separate PR, so we aren't mixing formatting changes with actual changes (otherwise it's a bit harder to review). Any thoughts?

Reply:
keen & happy to do, but can we merge these couple of MRs first in the interest of loooong running?